packages/teleport/src/DesktopSession/TdpClientCanvas/useTdpClientCanvas.tsx

```diff
@@ -141,6 +141,16 @@ export default function useTdpClientCanvas(props: Props) {
   };
 }
 
+// Calculates the size (in pixels) of the display.
+// Since we want to maximize the display size for the user, this is simply
+// the full width of the screen and the full height sans top bar.
+function getDisplaySize(): { width: number; height: number } {
+  return {
+    width: window.innerWidth,
+    height: window.innerHeight - TopBarHeight,
```

Comment:
It should be possible to get these dimensions without relying either on window or TopBarHeight. Canvas could be placed inside a div element where its dimensions will define the canvas size (you can use div.clientWidth and div.clientHeight). If you have time for this challenge, would be great to accept the challenge.

Reply:
I agree, but we are behind on session recording and so I'm going to focus on that for now.

Created an issue: https://github.com/gravitational/webapps/issues/485.